### DISCUSSION TITLE
cn/2020/02/21/just-do-it/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#165] [Announcements]
# cn/2020/02/21/just-do-it/

https://hongtaoh.com/cn/2020/02/21/just-do-it/

**Comment by Jacob-jinxy** [comment]
给楼主点赞，感觉你选对了路。
我申请的是商学院，极其hierarchical/demanding, research数学编程要求也高。准备申请这几年，什么兴趣/冲进都磨没了，job market还十分烂...
